### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by peppelinux**

### CODE REVIEW

**Review by selfissued — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+OpenID Federation 1.0 is:
+
+- a technical specification that shows us how to evaluate the Trust from a technical perspective.
+- a technology that shows us how to build trust infrastructures.
+- an advanced system for securely establishing the interoperability of participants.
+
+## Builds
+
+Previews for each branch of this project are automatically built and published at the URL *https://openid.github.io/federation/$branchname.html*.ù
+You can view the editor's copy at [https://openid.github.io/federation/main.html](https://openid.github.io/federation/main.html).
```
Comment (jogu) [inline README.md:17]: I suggest adding a link to the latest implementer's draft and latest working group version too.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Previews for branches associated with pending Pull Requests are also deployed and accessible using the same URL pattern.
+
+## Getting Started
+To get started with the OpenID Federation specification, clone this repository and review the documentation provided in the root directory in the form of xml files.
+
+````bash
+git clone https://github.com/openid-foundation/openid-federation.git
+cd openid-federation
+````
+Pleas read the [CONTRIBUTING.md](CONTRIBUTING.md) to learn how to contribute to this project.
```
Comment (jogu) [inline README.md:28]: typo: 

```suggestion
Please read the [CONTRIBUTING.md](CONTRIBUTING.md) to learn how to contribute to this project.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Contact
+For further information or to get involved, please visit the [OpenID Foundation website](https://openid.net/foundation/).
```
Comment (jogu) [inline README.md:31]: Suggest WG page instead or maybe as well:

```suggestion
For further information or to get involved, please visit the [OpenID Foundation website](https://openid.net/wg/connect/).
```

File: README.md (hunk 1 above)
Comment (peppelinux) [reply README.md:17]: good catch.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+The OpenID Federation specification outlines protocols and guidelines for implementing a federated infrastructure across different domains and services.
+
+## Overview
+
+OpenID Federation 1.0 is:
+
+- a technical specification that shows us how to evaluate the Trust from a technical perspective.
+- a technology that shows us how to build trust infrastructures.
+- an advanced system for securely establishing the interoperability of participants.
+
```
Comment (peppelinux) [inline README.md:13]:
```suggestion

## Status

All the news about OpenID Federation are available at the OpenID Foundation website, [here](https://openid.net/tag/federation/).

The latest official build of the specification is available at [https://openid.net/specs/openid-federation-1_0.html](https://openid.net/specs/openid-federation-1_0.html).
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+## Builds
+
+Previews for each branch of this project are automatically built and published at the URL *https://openid.github.io/federation/$branchname.html*.ù
```
Comment (peppelinux) [inline README.md:16]:
```suggestion
Previews for each branch of this project are automatically built and published at the URL *https://openid.github.io/federation/$branchname.html*.
```

File: README.md (hunk 1 above)
Comment (peppelinux) [reply README.md:22]: Done.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- a technical specification that shows us how to evaluate the Trust from a technical perspective.
+- a technology that shows us how to build trust infrastructures.
+- an advanced system for securely establishing the interoperability of participants.
+
+## Status
+
+All the news about OpenID Federation are available at the OpenID Foundation website, [here](https://openid.net/tag/federation/).
+
+The latest official build of the specification is available at [https://openid.net/specs/openid-federation-1_0.html](https://openid.net/specs/openid-federation-1_0.html).
```
Comment (peppelinux) [inline README.md:18]:
```suggestion
The latest official build of the specification is available at [https://openid.net/specs/openid-federation-1_0.html](https://openid.net/specs/openid-federation-1_0.html).

```

**Review by rohe — APPROVED** [review]

**Review by jogu — APPROVED** [review]

**Pull request merged**